features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP.cpp

```diff
@@ -57,6 +61,24 @@ static const intptr_t cellular_properties[AT_CellularBase::PROPERTY_MAX] = {
     0,  // PROPERTY_NON_IP_PDP_TYPE
     1,  // PROPERTY_AT_CGEREP
 };
+#else
+static const intptr_t cellular_properties[AT_CellularBase::PROPERTY_MAX] = {
```

Comment:
@AnttiKauppila Is this approach fine ? 

@wajahat-ublox Do we need to have this cellular_properties defined always?

Reply:
@0xc0170 The else condition is there for a cellular program using UBLOX APIs with an unsupported modem. Yes in case of non cellular programs, it should not be part of final binary.